Upgrade to Patternfly 6

tomasmatus opened the pull request:
Upgrade to PF6, use recent cockpit checkout, update other related dependencies

tomasmatus commented:
some screenshots to enjoy:

![Screenshot_20250429_162114](https://github.com/user-attachments/assets/2e945872-5f6b-4f21-b316-991d527cf489)
![Screenshot_20250429_162043](https://github.com/user-attachments/assets/60de8136-c762-41b6-9091-a0e25b9ef817)
![Screenshot_20250429_163157](https://github.com/user-attachments/assets/479230ff-330a-4265-9b4c-9bed2a0f08cf)
![Screenshot_20250429_162754](https://github.com/user-attachments/assets/ccbe8075-7b8a-46ff-bbfd-421d31214ba5)
![Screenshot_20250429_162334](https://github.com/user-attachments/assets/e1539c55-4384-4395-97d0-1ae786efdc9e)
![Screenshot_20250429_162134](https://github.com/user-attachments/assets/4d9e50c7-8bf3-4641-beff-86023c8cea4a)
![Screenshot_20250429_161953](https://github.com/user-attachments/assets/bf159187-04b5-4996-84e1-d9b5853b7660)

I wonder if I should so something about this inconsistency:
![Screenshot_20250429_162907](https://github.com/user-attachments/assets/ea317948-0ec9-4a19-9821-0c68b2cfe911)

Venefilyn commented:
> ![Screenshot_20250430_101836](https://private-user-images.githubusercontent.com/74668142/439140916-892e59b4-d9a7-4e33-8bf1-1e39a346df32.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDY1MzQxOTcsIm5iZiI6MTc0NjUzMzg5NywicGF0aCI6Ii83NDY2ODE0Mi80MzkxNDA5MTYtODkyZTU5YjQtZDlhNy00ZTMzLThiZjEtMWUzOWEzNDZkZjMyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA1MDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNTA2VDEyMTgxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNlYzU2Mzc3MWRhOWRlM2Y3Yzk0YTI3Yjg3ZjVlMmFlMDVkNjcxMjcxZmJmNWRlODExNThiMmYwYmY2OWMxY2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ZOVUjyF34IQ0QbUMQ24nTH-gkFTM0qBcYxWYhaaFaJA) 

This Alert should be inline and not have box shadow, as it is already inside a modal itself and not floating on the page

> I wonder if I should so something about this inconsistency: 
> ![Screenshot_20250429_162907](https://private-user-images.githubusercontent.com/74668142/439122127-ea317948-0ec9-4a19-9821-0c68b2cfe911.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDY1MzQxOTcsIm5iZiI6MTc0NjUzMzg5NywicGF0aCI6Ii83NDY2ODE0Mi80MzkxMjIxMjctZWEzMTc5NDgtMGVjOS00YTE5LTk4MjEtMGM2OGIyY2ZlOTExLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA1MDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNTA2VDEyMTgxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMwNDdjYzBhODVmM2RhODYzYWZjOTQwMmIwNGIwZWQyZjI3NzMwMjMzOGU4YzVmODJlYjU2OTc3MTBhOTU3ODcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.F9J6gpXB36fh0rLyDmB22uh4a01kTSn-3Yv9yoRiY5w)

IMO we shouldn't disable Cancel here, but maybe rename it to Close in that case and not disable it. Though for now this is fine

Venefilyn reviewed (approved):
Haven't had time to review it locally but code itself seems fine and if the UI looks good I'm fine to merge. It is better than how it was before at least

tomasmatus commented:
> This Alert should be inline and not have box shadow, as it is already inside a modal itself and not floating on the page

fixed

Also squashed and it is ready.

pkoprda reviewed (approved):
Thanks, I tried it locally and it works like a charm :+1: we can merge this
